themes/godotengine/layouts/features.htm

```diff
@@ -185,8 +185,8 @@ <h2>Multi-platform editor</h2>
       <h2>Multi-platform deploy</h2>
       <em>Deploy games everywhere!</em>
       <ul>
-        <li><b>Mobile platforms</b>, iOS, Android, BlackBerry OS</li>
-        <li><b>Desktop platforms</b> Windows (also UWP), OS X, Linux, *BSD and Haiku</li>
+        <li><b>Mobile platforms:</b> iOS, Android, BlackBerry OS</li>
+        <li><b>Desktop platforms:</b> Windows, UWP, MacOS, Linux, *BSD and Haiku</li>
```

Comment:
If we're splitting UWP from the main Windows mention, I'd put it after Linux (Linux, macOS and Windows are the main supported platforms - UWP is more beta, like *BSD and Haiku).

BTW macOS is spelled with a small "m" I think.